nsis/src/nsis/tests/NsisExecutableTest.java

```diff
@@ -32,10 +32,10 @@ public void testNsisCreationNotCompressed()
 			NsisExecutable ne = NsisExecutable.createNsisExecutable(
 					RethrowContinuesFactory.INSTANCE, bp, SectionLayout.FILE);
 
-			assertEquals(0x00008e04, ne.getHeaderOffset());
+			assertEquals(0x00008e00, ne.getHeaderOffset());
```

Comment:
out of scope for this PR, but something to do later: please add a failure test case, e.g. construct NsisExecutable out of garbage data, verify that correct exception is thrown.

Reply:
ref #18